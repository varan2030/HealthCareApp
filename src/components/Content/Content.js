import React, { Component } from 'react';

class Content extends Component {
  render() {
    return (
      <div className={ this.props.className }>
        { this.props.children }
      </div>
    )    
  }
}

export default Content