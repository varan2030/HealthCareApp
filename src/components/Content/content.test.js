import React from 'react';
import Enzyme, {shallow, mount} from 'enzyme';
import Content from "./";
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({adapter: new Adapter()});

describe("Content component", () =>{
  
  test('renders', () => {
    const wrapper = shallow(<Content />);
    expect(wrapper.exists()).toBe(true);
  })

  test('when the Content is clicked the Modal is opened', () =>{
    const wrapper = (<Content />);
    expect(wrapper).toMatchSnapshot();
  })
});
