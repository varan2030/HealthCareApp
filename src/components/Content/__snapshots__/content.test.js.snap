// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content component when the Content is clicked the Modal is opened 1`] = `<Content />`;
